src/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.Linux.cs

```diff
@@ -25,7 +25,7 @@ private ThreadPriorityLevel PriorityLevelCore
             }
             set
             {
-                throw new PlatformNotSupportedException();
+                throw new PlatformNotSupportedException(); // Why is this not supported? 
```

Comment:
Same as OSX, for the process thead priorities, I am not aware of a posix api to use..